### PR TITLE
fixed real device list issue

### PR DESCRIPTION
**Pull request by Shahnawaz-Sk**

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const request   = promisify(_request, Promise);
 
 const PROCESS_ENVIRONMENT = process.env;
 const BASE_URL = 'https://api.lambdatest.com/api/v1';
-const MOBILE_BASE_URL = 'https://beta-api.lambdatest.com/api/v1';
+const MOBILE_BASE_URL = 'https://mobile-api.lambdatest.com/mobile-automation/api/v1';
```
Comment (pawan-lambda) [inline src/util.js:15]: make sure this url is not used anywhere else, if yes verify change in reponse

**Review by pawan-lambda — APPROVED** [review]

File: src/util.js [hunk 2]
```diff
@@ -76,23 +76,25 @@ async function _getBrowserList () {
...
-    await axios.get(`${MOBILE_BASE_URL}/device?sort=brand&real=true`).then((res) => {
-        const iosDevices = res.data.ios;
+    await axios.get(`${MOBILE_BASE_URL}/real/list`).then((res) => {
 
-        const androidBrands = res.data.android;
-
-        const iosDeviceList = [];
-
-        const androidDeviceList = [];
+        var iosBrands = res.data.ios;
```
Comment (Shivanshu-lambdatest) [inline src/util.js:81]: what if res does not have data object it will through exeception

File: src/util.js [hunk 3]
```diff
@@ -76,23 +76,25 @@ async function _getBrowserList () {
...
+        var androidBrands = res.data.android;
```
Comment (Shivanshu-lambdatest) [inline src/util.js:82]: what if res does not have data object it will through exception

File: src/util.js [hunk 4]
```diff
@@ -76,23 +76,25 @@ async function _getBrowserList () {
...
+        var iosDeviceList = [];
+        var androidDeviceList = [];
+            
+        iosBrands.map((brand) => {
+            const iosDevices = brand.devices;
 
-        iosDevices.map((item) => {
-            const osVersion = item.osVersion;
+            iosDevices.map((device) => {
+                if (device.deviceType === 'real') {
```
Comment (Shivanshu-lambdatest) [inline src/util.js:90]: device?.deviceType

**Review by Shivanshu-lambdatest — APPROVED** [review]

**Pull request merged**